### DISCUSSION TITLE
<img> tag alt text contrast check

### DISCUSSION THREAD

**Discussion by pedropolidoro** [thedaviddias/Front-End-Checklist#526] [Ideas]
adding styling to image alt text in order to maintain contrast and readability. Optionally and less importantly for the purpose of maintaining design balance.
Without using` img { color: white }` it is impossible to read alternative text when the background-color is black.
![image](https://github.com/user-attachments/assets/ceaa5035-de00-492e-b734-ed0e273157d0)

**Comment by Musterihizmetleri** [comment]
Yorumlara dönüş yapan var mı
